catalog/opensearch/catalog-opensearch-source/src/main/java/ddf/catalog/source/opensearch/OpenSearchFilterVisitor.java

```diff
@@ -337,21 +346,26 @@ public Object visit(PropertyIsEqualTo filter, Object data) {
     public Object visit(PropertyIsLike filter, Object data) {
         logger.debug("ENTERING: PropertyIsLike filter");
 
-        LikeFilterImpl likeFilter = (LikeFilterImpl) filter;
+        if (curNest != NestedTypes.NOT) {
```

Comment:
Well its of type `NestedTypes`, so perhaps `currentNestedTypes` would be appropriate.